codespell_lib/data/dictionary_names.txt

```diff
@@ -6,5 +6,3 @@ liszt->list
 que->queue
 sargent->sergeant, argent,
 tim->time
-wen->when
```

Comment:
As per @qyearsley this is a name, if it's also in the main dictionary it needs removing from there.